doc/hooks/waf.section.md

```diff
@@ -20,10 +20,6 @@ If `wafPath` doesn't exist, then `wafHook` will copy the `waf` provided from Nix
 
 Controls the flags passed to waf tool during build and install phases. For settings specific to build or install phases, use `wafBuildFlags` or `wafInstallFlags` respectively.
 
-#### `dontAddWafCrossFlags` {#dont-add-waf-cross-flags}
```

Comment:
Hum, since it changes the documentation, should we include a note on release logs?

Reply:
Could do, but we don't have release notes for 24.11 yet.  I think that gets created as part of the release management process.